prometheus-exporters/install-exporter-amd64.sh

```diff
@@ -186,6 +200,97 @@ rules:
 EOF
   fi
 
+  if [ "$EXPORTER" == "opsverse-otelcontribcol" ]; then
+    cat << EOF > /etc/opsverse/exporters/opsverse-otelcontribcol/config.yaml
+receivers:
+  otlp:
+    protocols:
+      grpc:
+      http:
+  jaeger:
+    protocols:
+      grpc:
+      thrift_compact:
+      thrift_http:
+  zipkin: {}
+  # Dummy receiver that's never used, because a pipeline is required to have one.
+  # Picked highest port in attempt to avoid conflict
+  otlp/spanmetrics:
+    protocols:
+      grpc:
+        endpoint: "localhost:65535"
+processors:
+  batch:
+  spanmetrics:
+    metrics_exporter: prometheus
+    latency_histogram_buckets: [1ms, 2ms, 6ms, 10ms, 100ms, 250ms]
+    dimensions:
+      # If the span is missing http.method, the processor will insert
+      # the http.method dimension with value 'GET'.
+      # For example, in the following scenario, http.method is not present in a span and
+      # so will be added as a dimension to the metric with value "GET":
+      # - promexample_calls{http_method="GET",operation="/Address",service_name="shippingservice",\
+      #                     span_kind="SPAN_KIND_SERVER",status_code="STATUS_CODE_UNSET"} 1
+      - name: http.method
+        default: GET
+
+      # If a default is not provided, the http.status_code dimension will be omitted
+      # if the span does not contain http.status_code.
+      # For example, consider a scenario with two spans, one span having http.status_code=200 and another
+      # missing http.status_code. Two metrics would result with this configuration, one with the http_status_code
+      # omitted and the other included:
+      # - promexample_calls{http_status_code="200",operation="/Address",service_name="shippingservice",\
+      #                     span_kind="SPAN_KIND_SERVER",status_code="STATUS_CODE_UNSET"} 1
+      # - promexample_calls{operation="/Address",service_name="shippingservice",span_kind="SPAN_KIND_SERVER",\
+      #                     status_code="STATUS_CODE_UNSET"} 1
+      - name: http.status_code
+      # - service.name
+      # - operation
+      # - span.kind
+      # - status.code
+  memory_limiter:
+    # Same as --mem-ballast-size-mib CLI argument
+    ballast_size_mib: 683
+    # 80% of maximum memory up to 2G
+    limit_mib: 1500
+    # 25% of limit up to 2G
+    spike_limit_mib: 512
+    check_interval: 5s
+  probabilistic_sampler:
+    hash_seed: 22
+    sampling_percentage: 100
+  attributes/insert:
+    actions:
+      - key: "cluster"
+        value: "default-1"
+        action: insert
+extensions:
+  health_check: {}
+  zpages: {}
+exporters:
+  prometheus:
+    endpoint: "0.0.0.0:8889"
+  zipkin:
+    endpoint: "https://<tracesCollectorHost>/api/v2/spans"
+    headers:
+      'Authorization': 'Basic <base64EncodingOf devopsnow:pass>'
```

Comment:
Users might find this difficult and/or just throw in the wrong password.
Is it possible to accept both "tracesCollectorHost" as well as "devopsnow:pass" as inputs? That way we can just automatically fill up those values.

The prometheus remote write, I think is needed for the span metrics to be written to vmsingle. I'm surprised that some of the APOM metrics are showing up your grafana.

Reply:
That works and is better than the prometheus remote write from the collector.

In fact, we could do something very similar in our k8s agent too!

Reply:
Branch updated for the otel conf endpoints to be updated via CLI args (`-t` for the traces URL, `-u` for user, and `-p` for pass).

If either of them aren't passed, we'll prompt the user:

```
===============
No traces collector URL was passed in.
You can get this via the OpsVerse Admin Console:
  'Integrations' > 'URLs and Integrations' > 'Jaeger'
===============
Enter the traces collector URL (e.g., https://jane-doe.opsverse.cloud/api/v2/spans): https://<redacted>.observe.devopsnow.cloud/api/v2/spans
===============
No user for your traces collector endpoint was passed in.
You can get this via the OpsVerse Admin Console:
  'Integrations' > 'URLs and Integrations' > 'Jaeger'
===============
Enter the traces collector user (e.g., devopsnow): devopsnow
===============
No password for your traces collector endpoint was passed in. You can get this via the OpsVerse Admin Console:
  'Integrations' > 'URLs and Integrations' > 'Jaeger'
===============
Enter the traces collector password: <redacted>

```